xinference/model/llm/pytorch/core.py

```diff
@@ -224,6 +232,8 @@ def chat(
         full_prompt = self._to_prompt(prompt, system_prompt, chat_history=chat_history)
 
         generate_config = self._sanitize_generate_config(generate_config)
+        if "stop" not in generate_config and self._stop is not None:
```

Comment:
This should be part of `_sanitize_generate_config`

Reply:
done